### PR TITLE
[SDP-1184] Update the `db setup-for-network` CLI command to support Circle

### PR DESCRIPTION
**Pull request by ziyliu**
### What

Establish a different set of default assets for Circle distribution accounts.

### Why

The subset of Stellar assets supported by Circle is limited.

Address https://stellarorg.atlassian.net/browse/SDP-1184

#### PR Structure

* [x] This PR has a reasonably narrow scope (if not, break it down into smaller PRs).
* [x] This PR title and description are clear enough for anyone to review it.
* [x] This PR does not mix refactoring changes with feature changes (split into two PRs otherwise).

#### Thoroughness

* [x] This PR adds tests for the new functionality or fixes.
* [x] This PR contains the link to the Jira ticket it addresses.

#### Configs and Secrets

* [x] No new CONFIG variables are required -OR- the new required ones were added to the helmchart's [`values.yaml`] file.
* [x] No new CONFIG variables are required -OR- the new required ones were added to the deployments ([`pr-preview`], [`dev`], [`demo`], `prd`).
* [x] No new SECRETS variables are required -OR- the new required ones were mentioned in the helmchart's [`values.yaml`] file.
* [x] No new SECRETS variables are required -OR- the new required ones were added to the deployments ([`pr-preview secrets`], [`dev secrets`], [`demo secrets`], `prd secrets`).

#### Release

* [x] This is not a breaking change.
* [x] **This is ready for production.**. If your PR is not ready for production, please consider opening additional complementary PRs using this one as the base. Only merge this into `develop` or `main` after it's ready for production!

#### Deployment

* [x] Does the deployment work after merging?

[`values.yaml`]: ../helmchart/sdp/values.yaml
[`pr-preview`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/common-previews/stellar-disbursement-platform/backend-helm-values
[`dev`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/stellar-disbursement-platform/backend-helm-values
[`demo`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/stellar-disbursement-platform/demo/demo-backend-helm-values
[`pr-preview secrets`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/common-previews/externalsecrets-common-previews.yaml#L241-L346
[`dev secrets`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/stellar-disbursement-platform/stellar-disbursement-platform-externalsecrets.yaml
[`demo secrets`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/stellar-disbursement-platform/demo/demo-sdp-externalsecrets.yaml

### CODE REVIEW

File: internal/services/assets/assets_pubnet.go [hunk 1]
```diff
@@ -2,15 +2,27 @@ package assets
...
+const EURCAssetCode = "EURC"
+
 // USDCAssetCode is the code for the USDC asset for pubnet and testnet
 const USDCAssetCode = "USDC"
 
 // XLMAssetCode is the code for the XLM asset for pubnet and testnet
 const XLMAssetCode = "XLM"
 
+// EURCAssetIssuerPubnet is the issuer for the EURC asset for pubnet
+const EURCAssetIssuerPubnet = "GDHU6WRG4IEQXM5NZ4BMPKOXHW76MZM4Y2IEMFDVXBSDP6SJY4ITNPP2"
```
Comment (ziyliu) [inline internal/services/assets/assets_pubnet.go:15]: https://stellar.expert/explorer/public/asset/EURC-GDHU6WRG4IEQXM5NZ4BMPKOXHW76MZM4Y2IEMFDVXBSDP6SJY4ITNPP2

File: internal/services/assets/assets_testnet.go [hunk 2]
```diff
@@ -2,9 +2,18 @@ package assets
 
 import "github.com/stellar/stellar-disbursement-platform-backend/internal/data"
 
+// EURCAssetIssuerTestnet is the issuer for the EURC asset for testnet
+const EURCAssetIssuerTestnet = "GB3Q6QDZYTHWT7E5PVS3W7FUT5GVAFC5KSZFFLPU25GO7VTC3NM2ZTVO"
```
Comment (ziyliu) [inline internal/services/assets/assets_testnet.go:6]: https://stellar.expert/explorer/testnet/asset/EURC-GB3Q6QDZYTHWT7E5PVS3W7FUT5GVAFC5KSZFFLPU25GO7VTC3NM2ZTVO

File: internal/services/assets/assets_pubnet.go (hunk 1 above)
Comment (marcelosalloum) [reply internal/services/assets/assets_pubnet.go]: The official website promotes it here: https://www.circle.com/en/eurc

File: internal/services/assets/assets_testnet.go (hunk 2 above)
Comment (marcelosalloum) [reply internal/services/assets/assets_testnet.go]: The official website promotes it here: https://www.circle.com/en/eurc

**Review by marwen-abid — APPROVED** [review]
LGTM!

**Pull request merged**